.circleci/config.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       - yarn/setup
       - run:
           name: Danger
-          command: yarn danger ci
+          command: DANGER_GITHUB_API_TOKEN="15e52de81a772b174cc5""e1813d0083564c69c325" yarn danger ci
```

Comment:
what's up with this? is this..supposed to be here? cc @orta

Reply:
I just confirmed with @orta that this is ok. Danger doesn't get the token on forked PRs so it was failing.

Reply:
It's fine, it's because fork PRs don't get the access token from circle